documentation/modules/oauth/con-oauth-authentication-broker.adoc

```diff
@@ -45,8 +45,9 @@ spec:
       #...
 ----
 
-You can configure `plain`, `tls` and `external` listeners,
-but it is recommended not to use `plain` listeners or `external` listeners with disabled TLS encryption with OAuth 2.0 as this creates a vulnerability to network eavesdropping and unauthorized access through token theft.
+You can configure `plain`, `tls`, and `external` listeners.
+We recommend not using `plain` listeners or `external` listeners without TLS encryption enabled.
```

Comment:
This seems outdated as we do not have _`plain`, `tls`, and `external` listeners_ anymore. Maybe we could use this?

```suggestion
You can configure OAuth 2.0 authentication in your listeners.
We recommend using OAuth 2.0 authentication together with TLS encryption (`tls: true`).
```